src/Files.App/ViewModels/UserControls/SidebarViewModel.cs

```diff
@@ -271,6 +272,22 @@ private Task<LocationItem> CreateItemHomeAsync()
 			return CreateSectionAsync(SectionType.Home);
 		}
 
+		private void CreateItemSettingsAsync()
+		{
+			var settingsItem = new LocationItem()
```

Comment:
Can you set the tooltip to include the keyboard shortcut?

Reply:
Done